runner_benchmark/token_generator.py

```diff
@@ -81,7 +83,7 @@ def _get_payload_with_params(schema_name, schema_url=None, **extra_payload):
         datetime.now(tz=timezone.utc) + timedelta(days=7)
     ).isoformat()  # 7 days from now in ISO 8601 format
     for key, value in extra_payload.items():
-        payload_vars[key] = value
+        payload_vars['survey_metadata']['data'][key] = value
```

Comment:
This wouldn't currently work. Our current use for this is for top level data (schema_url).
You would need one for extra top level data and extra survey metadata.

Reply:
Ah okay, will update now - thanks